docs/source/examples/index.rst

```diff
@@ -0,0 +1,26 @@
+.. _examples:
+
+=========================
+Example Jupyter Notebooks
+=========================
+
+The following series of Jupyter Notebooks provide examples for usage of OpenMC
+features via the `Python API`_.
+
+.. _Python API: ../pythonapi/index.html
+
+.. toctree::
+    :maxdepth: 1
+
+    post-processing
+    pandas-dataframes
+    tally-arithmetic
+    mgxs-part-i
+    mgxs-part-ii
+    mgxs-part-iii
```

Comment:
If that's the case, then you should remove mgxs-part-iv.ipynb and mgxs-part-iv.rst